python_scripts/customize_octopi_config.py

```diff
@@ -14,6 +14,14 @@ def replace_line_starting_with(starting_with: str, replacement: str, string: str
     return re.sub(rf"^[ #]*{starting_with}.*$", replacement, string, count=1, flags=re.MULTILINE)
 
 
+# Downloading updated version of octopi.txt
+with urlopen('https://raw.githubusercontent.com/guysoft/OctoPi/devel/src/modules/octopi/filesystem/boot/octopi.txt') as file_response:
+    data = file_response.read().decode()
+
+# Writing new version of octopi.txt to file
+with open('/boot/octopi.txt', 'w') as file:
+    file.write(data)
```

Comment:
It could improve the code of this file as a whole if you integrated this with the logic below - mainly the check for whether it's necessary to actually write to the file or not (`if new_file_contents != file_contents`) 🙂 I should have added a comment to that check, stating the reason why it's there, but oh well 😅

Reply:
I've now rewritten the script, so that the replacement strings are replaced in the file downloaded from GitHub. Then the old and the new version are compared, and if they are different, the new version is written to file.